content/handbook/security/best-practices/google-cloud-security-best-practices.md

```
@@ -0,0 +1,132 @@
+
+このテストまたはデモ環境がかなりの期間（24 時間以上）稼働している場合は、廃止日を計画し、適時にインスタンスがシャットダウンまたは破棄されるようにリマインダーを設定してください。
+
+#### 定期的にアップグレードする
+
+長期間稼働するテストまたはデモインスタンスの場合、最新のセキュリティパッチがすべて適用されていることを確認するために、GitLab とマシン上の他のソフトウェアを定期的に更新することが重要です。
+
+すべてのセキュリティリリース後に GitLab を更新する計画を立て、システムパッケージを少なくとも週に 1 回更新してください。
+
+GitLab セキュリティ更新を適時に適用することを確認するために、自分自身にリマインダーを設定してください。
+
+また、テストまたはデモが完了したらすぐにインスタンスを [クリーンアップ](#clean-up) することを忘れないでください。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)